test/units/testsuite-19.cleanup-slice.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
hmm, this just drops the shell script in to test-19, but it's not actually started, is it? what am i missing?

Reply:
`run_subtests` in `testsuite-19.sh` executes all the scripts `testsuite-19.*.sh` subscripts

https://github.com/systemd/systemd/blob/b58feca65cb2426b553828e5dff2eb0b213fa05b/test/units/testsuite-19.sh#L11